package.json

```diff
@@ -33,5 +33,22 @@
     "response-time": "^2.3.2",
     "sweet-feedback": "^1.0.1",
     "textangular": "^1.5.16"
+  },
+  "devDependencies": {
+    "@typescript-eslint/experimental-utils": "^3.1.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jest": "^23.13.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.3",
+    "eslint-plugin-react": "^7.20.0",
+    "eslint-plugin-react-hooks": "^2.5.0",
+    "prettier": "^2.0.5",
+    "typescript": "^3.9.3"
```

Comment:
Is it necessary the typescript dependency here?

Reply:
it was necessary for an airbnb dependency, but I changed it to airbnb-base, and it is no longer needed